.github/workflows/helm_ci.yaml

```diff
@@ -0,0 +1,70 @@
+name: ci-tests
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'charts/**'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint-test-chart:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s_version:
+          - v1.31.0
+          - v1.30.0
+          - v1.29.0
```

Comment:
If we are going to tests those versions, pronably we should also limit the versions supported on Chart.yaml.

Reply:
@tamcore  I think you didn't get my point. I'm trying to say that we should update the chart.yaml to 1.29.0 min version as part of this PR. With your PR, we are kinda supporting it better.

Reply:
here: https://github.com/Koenkk/zigbee2mqtt-chart/blob/405f3e460c9f0250709efc956e2ffc6e898befe2/charts/zigbee2mqtt/Chart.yaml#L6

Reply:
I still think makes sense since this is what we are supporting.  The same GVK adds some fields in some version (sometimes by gates), I think it make sense that we test what we suport and we put on the manifests what we support.